spark-common/src/main/js/src/global.js

```diff
@@ -0,0 +1,27 @@
+import spark from './index';
+
+function getSparkFail() {
+    throw new Error('getSpark() has to be called immediately after including spark-dist.js!');
+}
+
+/**
+ * Get the last included version of Spark. Has to be called immediately after including spark-dist.js (to avoid another version of SPARK
+ * to be included in the meantime).
+ * @param allowMultipleCalls {Boolean} Allow getSpark() to be called once more (useful to set if you have multiple JS files that are
```

Comment:
Do we really need this parameter?
See comment in ```content_iframe_wrapper.vm```

This is kind of complicated to use because the last file accessing spark would need to call the method without the parameter.
If a plugin really has several Javascripts that need access to spark it can always declare its own global variable with a unique name and assign the return value of ```window.getSpark()``` to that one. That approach would also work if ```spark-dist.js``` is pulled in through a separate ```<web-resource>``` module.